docs/advanced/pycpp/object.rst

```diff
@@ -62,6 +64,40 @@ This example obtains a reference to the Python ``Decimal`` class.
     py::object scipy = py::module::import("scipy");
     return scipy.attr("__version__");
 
+.. _implicit_casting:
+
+Implicit (automatic) casting
```

Comment:
Let's get rid of "automatic", here, maybe? I'm not sure if it helps and it is probably implicit in "implicit".